src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -157,7 +158,7 @@ function addGlobalModuleScriptTagsToDocument(callback: () => void) {
 
 function createEmscriptenModuleInstance(resourceLoader: WebAssemblyResourceLoader, onReady: () => void, onError: (reason?: any) => void) {
   const resources = resourceLoader.bootConfig.resources;
-  const module = {} as typeof Module;
+  const module = (window['Module'] || { }) as typeof Module;
```

Comment:
This plays slightly better with other scripts that might want to hook up to `Module.preRun`. The initial timezone js file that was shared by the runtime team relied on being able to add a `preRun` task.